packages/terra-button-group/src/ButtonGroup.jsx

```diff
@@ -0,0 +1,143 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+import ButtonGroupButton from './ButtonGroupButton';
+import './ButtonGroup.scss';
+
+const propTypes = {
+  /**
+   * Callback function when the state changes
+   **/
+  onChange: PropTypes.func,
+
+  /**
+   * Sets the button group style variation. One of `defaut` or `secondary`
+   **/
+  variant: PropTypes.oneOf(['default', 'secondary']),
```

Comment:
No, the UX standards only wanted default and secondary to allowed styles for the button group